.github/workflows/cypress_tests.yml

```diff
@@ -87,131 +86,14 @@ jobs:
           docker pull $ECR_REGISTRY/$ECR_REGISTRY_ALIAS/$IMAGE_NAME:$IMAGE_TAG
           docker tag $ECR_REGISTRY/$ECR_REGISTRY_ALIAS/$IMAGE_NAME:$IMAGE_TAG $IMAGE_NAME:$IMAGE_TAG
 
-      - name: Run Cypress Tests - @Signup,@StitchedPF
-        env:
-          CYPRESS_adminUrl: ${{ inputs.admin_url }}
-          CYPRESS_baseUrl: ${{ inputs.front_url }}
-          CYPRESS_NO_COMMAND_LOG: 1
-          CYPRESS_numTestsKeptInMemory: 1
-          CYPRESS_RUNNER_IN_CI: true
-          CYPRESS_RUNNER_TAGS: "@Signup,@StitchedPF"
-          CYPRESS_RUNNER_BASE_URL: ${{ inputs.front_url }}
-          CYPRESS_RUNNER_ADMIN_URL: ${{ inputs.admin_url }}
-
-        run: |
```

Comment:
Have you deliberately removed all of the run definitions? Do you need one of these to be present for anything to happen? Or is this just to test the principle without actually doing anything?

Reply:
This may be easier to see here: https://github.com/ministryofjustice/opg-lpa/blob/483b114e9c162f324a921e080cea77c309b853a6/.github/workflows/cypress_tests.yml

There's only one Cypress step now but the entire job gets called multiple times with the different tags